docs/source/markdown/podman-exec.1.md

```diff
@@ -43,7 +43,21 @@ Pass down to the process N additional file descriptors (in addition to 0, 1, 2).
 
 **--privileged**
 
-Give the process extended Linux capabilities when running the command in container.
+Give extended privileges to this container. The default is *false*.
+
+By default, Podman containers are
+“unprivileged” and cannot, for example, modify parts of the kernel.
+This is because by default a container is not allowed to access any devices.
+A “privileged” container is given access to all devices that the user creating
```

Comment:
A "privileged" container is given the same access to devices as the user launching the container.